<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>-1</y>
     <width>811</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Установка данных</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>781</width>
       <height>531</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="title">
           <string>Загрузить датасет</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>19</y>
             <width>371</width>
             <height>114</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="datasetCombo"/>
            </item>
            <item>
             <widget class="QPushButton" name="datasetLoad">
              <property name="text">
               <string>Загрузить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="datasetPreview">
              <property name="text">
               <string>Предпросмотр</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="datasetDelete">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Сгенерировать датасет</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>19</y>
             <width>371</width>
             <height>331</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Начальное положение</string>
              </property>
              <widget class="QWidget" name="gridLayoutWidget_2">
               <property name="geometry">
                <rect>
                 <x>-1</x>
                 <y>19</y>
                 <width>371</width>
                 <height>51</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="1">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Y</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>X</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QSpinBox" name="x0">
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>27</number>
                  </property>
                  <property name="singleStep">
                   <number>1</number>
                  </property>
                  <property name="value">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="y0">
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>27</number>
                  </property>
                  <property name="value">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Финальное положение</string>
              </property>
              <widget class="QWidget" name="gridLayoutWidget_3">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>20</y>
                 <width>371</width>
                 <height>51</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="1">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Y</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>X</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QSpinBox" name="x1">
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>27</number>
                  </property>
                  <property name="value">
                   <number>27</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="y1">
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>27</number>
                  </property>
                  <property name="value">
                   <number>27</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="1">
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Скорость движения, пкс/с</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="speed">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>1000000</number>
                </property>
                <property name="value">
                 <number>1000</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QSpinBox" name="radius">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>5</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Радиус мишени</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="previewNew">
              <property name="text">
               <string>Предпросмотр</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="appendToExisting">
                <property name="text">
                 <string>Добавить к существующему</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="saveAsNew">
                <property name="text">
                 <string>Сохранить как новый</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="image_container">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="preview">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>381</width>
           <height>521</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Обучение</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>781</width>
       <height>541</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="chooseConf">
          <property name="text">
           <string>Выбрать файл с конфигурацией модели</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="setsGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Выбрать датасеты для обучения</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>19</y>
             <width>371</width>
             <height>421</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="setsContainer"/>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Сколько показать датасетов</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="letsGo">
            <property name="text">
             <string>Начать обучение</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="numOfSets">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10009</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Размер пула</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="poolSize">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="image_container_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="trainResult">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>381</width>
           <height>521</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Распознание</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>771</width>
       <height>541</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="chooseModel">
          <property name="text">
           <string>Выбрать модель</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="setsGroupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Выбранные датасеты</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_5">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>19</y>
             <width>361</width>
             <height>391</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="reckonSetsContainer"/>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="addSet">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="deleteSets">
          <property name="text">
           <string>Удалить отмеченные</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reckon">
          <property name="text">
           <string>Распознать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="reckonGraphics">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="answers">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="statusbar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>791</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
